eclair-core/src/main/scala/fr/acinq/eclair/channel/fund/InteractiveTxBuilder.scala

```diff
@@ -451,7 +451,10 @@ private class InteractiveTxBuilder(replyTo: ActorRef[InteractiveTxBuilder.Respon
     session.toSend match {
       case (addInput: Input) +: tail =>
         val message = addInput match {
-          case i: Input.Local => TxAddInput(fundingParams.channelId, i.serialId, Some(i.previousTx), i.previousTxOutput, i.sequence)
+          case i: Input.Local =>
+            // for debugging wallet locking issues, it is useful to log local utxos
+            log.info(s"adding local input ${i.previousTx.txid}:${i.previousTxOutput}")
```

Comment:
Shouldn't we add a bit more context about what we're doing? We don't have any previous log mentioning that we're inside an interactive-tx session.

```suggestion
            log.info(s"adding local input ${i.previousTx.txid}:${i.previousTxOutput} to interactive-tx")
```

Reply:
The context is already provided in the logger name:

```
15:25:53.724 INFO  f.a.e.c.f.InteractiveTxBuilder n:02eeee c:de2e6ded - adding local input c0642e334e9a146dd65110a619a78ecde14da999ff7a298a3d1c507007cbe8fa:0
```